src/assets/skins/lao.scss

```diff
@@ -0,0 +1,9 @@
+.lao {
```

Comment:
The skin file should be linked here: https://github.com/balancer-labs/snapshot/blob/develop/src/assets/skins/index.scss

Reply:
ah, good point. will stitch in~~

Reply:
ok, I think I was able to resolve :+1: @bonustrack thanks for quick lookover!